rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -5207,11 +5207,11 @@ void Compiler::CreateBranch(llvm::Value * cmp_i1, llvm::Value * target_i32, bool
 			}
 
 			SetPc(target_i32);
-//			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
+			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
```

Comment:
Yes, found this for insertion this function
https://github.com/RPCS3/rpcs3/commit/eb90e509f9845d2c89acf63cd4e1377f26872c89